app/actions/configurations.py

```diff
@@ -17,6 +17,13 @@ class AuthenticateConfig(AuthActionConfiguration, ExecutableActionMixin):
     )
 
 
+class ProcessEventsPerAOIConfig(PullActionConfiguration):
```

Comment:
Use `InternalActionConfiguration` for sub-actions, unless you need some user input and you want the config to show up in the portal:
```suggestion
class ProcessEventsPerAOIConfig(InternalActionConfiguration):
```